__tests__/app.test.js

```diff
@@ -92,6 +92,53 @@ describe("/api/articles", () => {
             })
         })
     })
+    describe("GET Queries", () => {
+        test("?sort_by=: Responds with all article objects ordered by the column of the given 'sort_by' query", () => {
+            return request(app)
+            .get("/api/articles?sort_by=title")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles).toHaveLength(13)
+                expect(body.articles).toBeSortedBy('title', {descending: true})
+            })
+        })
+        test("?sort_by= 400: Responds with 'Bad request' when the given column name doesn't exist in the table", () => {
+            return request(app)
+            .get("/api/articles?sort_by=not-a-column")
+            .expect(400)
+            .then(({body}) => {
+                expect(body).toEqual({message: "Bad request"})
+            })
+        })
+        test("?order=: Responds with all article objects ordered by the given 'order' query", () => {
+            return request(app)
+            .get("/api/articles?order=asc")
+            .expect(200)
+            .then(({body})=> {
+                expect(body.articles).toHaveLength(13)
+                expect(body.articles).toBeSortedBy('created_at')
```

Comment:
I think because of the test, I'd expect to see the order asc be true as well :)

Reply:
@nc-niamh-smith 
Cheers Niamh, Isn't it ascending by default?

Reply:
Yessss but it'd be quite nice to have it explicit in case anyone looking wasn't 100%! :)

Reply:
Thats fair. Cheers :)